src/HelpersServiceProvider.php

```diff
@@ -24,6 +27,40 @@ public function boot()
         app(ExcelServiceProvider::class, ['app' => app()])->boot();
 
         $this->loadViewsFrom(__DIR__ . '/Stubs', 'ronasit');
+
+        /**
```

Comment:
let's move it to the separate function `extendRouter`

Reply:
Done.

Reply:
please move only router extension to the separate method, not all logic :)

Reply:
Done.